cmd/clusterctl/examples/openstack/README.md

```diff
@@ -18,31 +18,51 @@
 ## Prerequisites
 
 1. Install `yq` (see [here](https://github.com/mikefarah/yq)).
+2. Install `kustomize` v1.0.11 which can be found [here](https://github.com/kubernetes-sigs/kustomize/releases/tag/v1.0.11).
 
 ## Generation
 For convenience, a generation script which populates templates based on openstack cloud provider
 configuration is provided.
 
 1. Run the generation script.
 ```
-./generate-yaml.sh --provider-os [os name]
+./generate-yaml.sh [options] <path/to/clouds.yaml> <openstack cloud> --provider-os [centos,ubuntu,coreos]
```

Comment:
./generate-yaml.sh  clouds.yaml openstack --provideros coreos
Error: No provider OS specified
You must choose between the following operating systems: centos, ubuntu, coreos


root@kvm-017751:~/go/src/sigs.k8s.io/cluster-api-provider-openstack/cmd/clusterctl/examples/openstack# ./generate-yaml.sh  clouds.yaml openstack coreos
root@kvm-017751:~/go/src/sigs.k8s.io/cluster-api-provider-openstack/cmd/clusterctl/examples/openstack#

Reply:
yes, replaced the `--provider-os` to `<provider os>`.